### ISSUE TITLE
refresh() reports the wrong elapsed time

### ISSUE THREAD

**Issue by ferrouswheel** [tqdm/tqdm#269]
v4.8.4

```python
import time
from tqdm import tqdm, trange
bar = tqdm(total=10)

for i in range(0,10):
	bar.update(1) # reports i*2 seconds elapsed

	time.sleep(1)
	bar.refresh() # reports 00:01 seconds elapsed
	time.sleep(1)
```

This line https://github.com/tqdm/tqdm/blob/master/tqdm/_tqdm.py#L649 in `__repr__` should probably use `self._start_time`?

A work around is using `update(0)` as long as you avoid #268

**Comment by ferrouswheel** [comment]
Sorry for the flurry of issues. I really like tqdm so I'm probably putting it through its paces and doing some stuff that not many people try.

**Comment by ferrouswheel** [comment]
Using `tqdm.write()` will exhibit the same behaviour as it calls `refresh()`

**Comment by lrq3000** [comment]
No that's good Joel, we need testers. Thank you for reporting the issues, I
will take a look later (finishing some PR right now).

2016-09-14 3:44 GMT+02:00 Joel <notifications@github.com>:

**Comment by lrq3000** [comment]
Good catch, this will be fixed. Thank's!

**Comment by lrq3000** [comment]
Closed by #264.
